doc/classes/StreamPeerTCP.xml

```diff
@@ -61,7 +61,7 @@
 			<argument index="0" name="enabled" type="bool">
 			</argument>
 			<description>
-				Disables Nagle's algorithm to improve latency for small packets.
+				When enabled, packets will be sent immediately; otherwise, packet transfers will be delayed and combined using Nagle's Algorithm. 
```

Comment:
```suggestion
				If [code]enabled[/code] is [code]true[/code], packets will be sent immediately. If [code]enabled[/code] is [code]false[/code] (the default), packet transfers will be delayed and combined using [url=https://en.wikipedia.org/wiki/Nagle%27s_algorithm]Nagle's algorithm[/url].
```

Reply:
Can't find any concrete evidence to support it being disabled by default, but [this SO post](https://stackoverflow.com/a/31997755/7260220) implies it.

> You should turn this option on only if you really know what you are doing.